src/lua/uri.lua

```diff
@@ -227,4 +306,12 @@ return {
         params = params,
         encode_kv = encode_kv,
     },
+    str_escape = str_escape,
+    str_unescape = str_unescape,
```

Comment:
I like `uri.escape()` and `uri.unescape()` names we had during discussing a future implementation. If you don't mind, I would leave them.

Reply:
So we both want to change something we agreed upon before. Let's discuss within the working group (at least engage @Mons).